_data/past.yml

```diff
@@ -1142,3 +1142,15 @@
   dates: "May 25-26, 2018"
   url: http://balkanruby.com/
   twitter: balkanruby
+
+- name: RubyC
+  location: Kyiv, Ukraine
+  dates: "June 2-3, 2018"
+  url: http://rubyc.eu/
+  twitter: rubyc_eu
+
+- name: RubyKaigi 2018
+  location: Sendai, Japan
+  dates: "May 31st-June 2nd, 2018"
```

Comment:
I know that this is a copy-and past, but do you mind removing `st` and `nd` just so it's in the same format as other conferences?